pywbem/_cim_operations.py

```diff
@@ -848,6 +859,27 @@ def __init__(self, url, creds=None, default_namespace=None,
                 detail_levels=self._log_detail_levels)
             self.add_operation_recorder(recorder)
 
+    def __enter__(self):
```

Comment:
One more comment.  Should we mention that this class is context aware in the general documentation about the class?

Other than that question, approved

Reply:
The docstring of class WBEMConnection already does that, even with an example, on line 383 of _cim_operations.py:
```
    The WBEMConnection class can also be used as a context manager, which
    causes the connection to be closed at context manager exit:

    .. code-block:: python

        with WBEMConnection('https://myserver') as conn:
            conn.EnumerateInstances('CIM_Foo')
```